lib/proxywrap.js

```diff
@@ -290,5 +281,21 @@ exports.proxy = function(iface, options) {
 
 	}
 
+	function createSecureConnectionListener(context, listener) {
+		return function (socket) {
+			var properties = Object.keys(proxyProtocolFields);
+			defineTLSSocketProperties(socket, properties);
+			listener.call(context, socket);
+		}
+	}
+
+	function defineTLSSocketProperties(tlsSocket, properties) {
+		for(var i in properties) {
```

Comment:
It would be nice if you add the change to the [CHANGELOG](https://github.com/findhit/proxywrap/blob/master/CHANGELOG.md) under "upcoming".